Tests/js/views/services/plugins/assets/ez-publishdraftplugin-tests.js

```diff
@@ -109,20 +114,50 @@ YUI.add('ez-publishdraftplugin-tests', function (Y) {
             delete this.parentLocation;
             delete this.contentType;
         },
+        _onNotification: function (that) {
```

Comment:
empty line before. And why passing the object which also the context of execution of the method (*this*) ?